tests/runtest.sh

```diff
@@ -524,6 +528,14 @@ function run_test {
     local scriptFileName=$(basename "$scriptFilePath")
     local outputFileName=$(basename "$outputFilePath")
 
+    if [ ! "$noLFConversion" == "ON" ]; then
+        # Convert DOS line endings to Unix if needed
+        perl -pi -e 's/\r\n|\n|\r/\n/g' "$scriptFileName"
+    fi
+
+    # Add executable file mode bit if needed
+    chmod +x "$scriptFileName"
```

Comment:
It looks like this was moved elsewhere by a recent change, and both copies were kept after your merge. Can you remove it?